include/decoder.hpp

```diff
@@ -7,14 +7,15 @@
  *   Tassilo Tanneberger
  */
 
-#ifndef TETRA_DECODER_DECODER_HPP
-#define TETRA_DECODER_DECODER_HPP
+#pragma once
```

Comment:
not standard include guards?

Reply:
as discussed we now use `#pragma once` everywhere